src/common/snippets/tests/src/pass/softmax_reshape_elimination.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ngraph/function.hpp>
+#include <ngraph/pass/manager.hpp>
+
+#include <snippets/snippets_isa.hpp>
+#include <snippets/pass/softmax_reshape_elimination.hpp>
+
+#include <transformations/init_node_info.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+using namespace ngraph;
+
+TEST_F(TransformationTestsF, SoftmaxV1ReshapeElimination) {
+    {
+        auto data = std::make_shared<opset1::Parameter>(element::f32, Shape{2, 3, 240});
+        auto shape0 = std::make_shared<ov::op::v0::Constant>(ov::element::i32, ov::Shape{2}, std::vector<int32_t>{6, 240});
+        auto reshape0 = std::make_shared<ov::op::v1::Reshape>(data, shape0, false);
+        auto softmax_v1 = std::make_shared<ov::op::v1::Softmax>(reshape0, 1);
+        auto shape1 = std::make_shared<ov::op::v0::Constant>(ov::element::i32, ov::Shape{3}, std::vector<int32_t>{2, 3, 240});
+        auto reshape1 = std::make_shared<ov::op::v1::Reshape>(softmax_v1, shape1, false);
+        function = std::make_shared<Function>(NodeVector{reshape1}, ParameterVector{data});
+
+        manager.register_pass<pass::InitNodeInfo>();
```

Comment:
Minor: this pass is added by default in `TransformationTestsF::SetUp()` method so we can remove it

Reply:
Removed, thanks!